doc/nrf/index.rst

```diff
@@ -3,6 +3,17 @@
 Welcome to the |NCS|!
 #####################
 
+.. important::
+
+   |NCS| v1.9.99-dev1 is a development tag and will be replaced by v2.0.0 in the future.
+
+   The development tag contains the current development state for LwM2M 1.1 and AVSystem integration for nRF Cloud Location Services through LwM2M, which is considered experimental as of now.
+
+   The development tag introduces a Bluetooth LE Audio standard example through the :ref:`octave` application, which uses the new LE Audio Controller Subsystem for nRF53.
+   Both the application and the subsystem are marked as experimental.
+
+   For other changes that are included in this development tag, see :ref:`ncs_release_notes_changelog`.
```

Comment:
```suggestion
.. important::
   The |NCS| v1.9.99-dev1 is a development tag and will be replaced by v2.0.0 in the future.

   The development tag comes with the following major changes, marked as experimental:
   
   * The current development state for LwM2M v1.1 and AVSystem integration for nRF Cloud Location Services through LwM2M.
   * An example of the Bluetooth LE Audio standard through the :ref:`octave` application, which uses the new LE Audio Controller Subsystem for nRF53.

   For other changes that are included in this development tag, see :ref:`ncs_release_notes_changelog`.
```
I feel we could save some space with these edits.

Reply:
Updated, as agreed.